src/patternfly/components/Select/docs/select-checkbox.md

```diff
@@ -0,0 +1,32 @@
+The checkbox select can select multiple items using checkboxes. The number of items selected is reflected in a badge in the dropdown toggle. The user may clear items by unchecking or using the clear button. Optionally, items may be grouped.
+
+## Accessibility
+
+*This section to be updated once the React implementation is complete.*
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `role` or `aria` | `pf-c-select` |  accessibility notes. |
+*Note:* The attribute `aria-pressed="true"` should be set programmatically to the active item.
+
+
+## Usage
+
+| Class | Applied To | Outcome |
```

Comment:
Looking at the [table](https://pf-next.com/components/Table/examples/) as an example, it looks like we're not documenting things that were already covered in previous docs, so I wonder if we can remove rows from the docs that have already been covered in a previous example?

Reply:
I have mixed feelings about this - definitely don't like duplicating documentation, but at the same time, this is integral to the component at hand and I like seeing it all listed so that I can understand the entire component structure.

Reply:
I agree. That the individual examples only document the changes in that example from the other examples is one of the reasons I'm not a fan of writing it that way. You have to go read other examples to get the rest of the docs that are relevant to whatever example you're looking at. Though duplicating the lines is isn't good, either, since it's just copy/paste. Maybe we can discuss in our next CSS meeting?